providers/standard/src/airflow/providers/standard/sensors/date_time.py

```diff
@@ -99,6 +98,15 @@ def poke(self, context: Context) -> bool:
         self.log.info("Checking if the time (%s) has come", self.target_time)
         return timezone.utcnow() > timezone.parse(self.target_time)
 
+    @property
+    def _moment(self) -> datetime.datetime:
+        import pendulum
+
+        if isinstance(self.target_time, (pendulum.DateTime, datetime.datetime)):
```

Comment:
```suggestion
        if isinstance(self.target_time, datetime.datetime):
```

Pendulum DateTime is stdlib datetime.

Reply:
Once this comment has been resolved, I think we can merge it 🙂

Reply:
You're right, `pendulum` isn't necessary here; I'll remove it.

Reply:
fixed, many thanks 😄